mirrord/agent/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM --platform=$BUILDPLATFORM ghcr.io/metalbear-co/ci-agent-build:latest as bui
 ARG TARGETARCH
 WORKDIR /app
 
+RUN apt-get install -y protobuf-compiler
```

Comment:
add it in the base image in the ci repository

Reply:
metalbear-co/ci#3